src/rngs/os.rs

```diff
@@ -147,6 +147,7 @@ impl RngCore for OsRng {
 
 #[cfg(all(unix,
           not(target_os = "cloudabi"),
+          not(all(target_os = "emscripten", feature = "stdweb")),
```

Comment:
This doesn't check the architecture, but the other impl checks the architecture. This leaves a hole where no impl is compiled in.